### PR TITLE
Remove override for argmax/argmin

### PR DESCRIPTION
**Pull request by ailzhang**
#1225

### CODE REVIEW

File: test/cpp/test_aten_xla_tensor.cpp [hunk 1]
```diff
@@ -1831,6 +1831,8 @@ TEST_F(AtenXlaTensorTest, TestArgMin) {
     torch::Tensor xla_b = torch::argmin(xla_a, c10::nullopt, /*keepdim=*/false);
     AllEqual(b, xla_b);
   });
+
+  ExpectCounterNotChanged("aten::.*", cpp_test::GetIgnoredCounters());
```
Comment (dlibenzi) [inline test/cpp/test_aten_xla_tensor.cpp]: I think we need to start asserting for things we want to see?

https://github.com/pytorch/xla/pull/1250

Like:
```Python
ExpectCounterNotChanged("aten::.*", cpp_test::GetIgnoredCounters());
ExpectCounterChanged("xla::...", cpp_test::GetIgnoredCounters());
```

File: torch_xla/csrc/aten_xla_type.cpp [hunk 2]
```diff
@@ -511,24 +511,6 @@ at::Tensor& AtenXlaType::arange_out(at::Tensor& out, at::Scalar start,
...
-                   XLATensor::argmax(bridge::GetXlaTensor(self), *dim, keepdim))
-             : bridge::AtenFromXlaTensor(
-                   XLATensor::argmax(bridge::GetXlaTensor(self)));
-}
-
-at::Tensor AtenXlaType::argmin(const at::Tensor& self,
-                               c10::optional<int64_t> dim, bool keepdim) {
-  XLA_FN_COUNTER("xla::");
-  return dim ? bridge::AtenFromXlaTensor(
-                   XLATensor::argmin(bridge::GetXlaTensor(self), *dim, keepdim))
```
Comment (dlibenzi) [inline torch_xla/csrc/aten_xla_type.cpp]: I don't see XLATensor argmin/argmax removed from there.
Are there other call sites?

File: test/cpp/test_aten_xla_tensor.cpp (hunk 1 above)
Comment (ailzhang) [reply test/cpp/test_aten_xla_tensor.cpp]: Hmmm I'm afraid adding positive assert for non leaves are too much. I agree it's good to add for leaves. This is a moving part in Pytorch, I feel asserting that will force us to change tests very frequently?

File: torch_xla/csrc/aten_xla_type.cpp (hunk 2 above)
Comment (ailzhang) [reply torch_xla/csrc/aten_xla_type.cpp]: Done! :D

File: test/cpp/test_aten_xla_tensor.cpp (hunk 1 above)
Comment (ailzhang) [reply test/cpp/test_aten_xla_tensor.cpp]: Nvm, let's add them now and delete in the future if too much noise ;) 
Sometimes it's good to get a signal when the lowered graph changes as well. Hopefully it doesn't happen often.

Comment (dlibenzi) [reply test/cpp/test_aten_xla_tensor.cpp]: Yes. The point is, if today we assume pytorch does a certain lowering, and tomorrow it changes to a different one which is less performant, we better know.

**Pull request merged**